### PR TITLE
[SW-213] Stopped ignoring FrameTreeSnapshot proto

### PR DESCRIPTION
**Pull request by amessing-bdai**

### CODE REVIEW

File: msg/KeyValueBosdynMsgsSerializedMessage.msg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# AUTOMATICALLY GENERATED BY PROTO CONVERSION
+
+ key
```
Comment (jbarry-bdai) [inline msg/KeyValueBosdynMsgsSerializedMessage.msg:3]: This isn't right...

**Review by jbarry-bdai — COMMENTED** [review]
FrameTreeSnapshot message is missing?

CMakeLists.txt should also need to be updated from the FrameTreeSnapshot message.

File: msg/KeyValueBosdynMsgsSerializedMessage.msg (hunk 1 above)
Comment (amessing-bdai) [reply msg/KeyValueBosdynMsgsSerializedMessage.msg:3]: I was curious about that

**Comment by amessing-bdai** [comment]
https://github.com/bdaiinstitute/bosdyn_msgs/blob/amessing/frametreesnapshot/msg/FrameTreeSnapshot.msg

It previous existed, so wasn't created or editted. We were simply ignoring it as a parameter

File: msg/KeyValueBosdynMsgsSerializedMessage.msg (hunk 1 above)
Comment (amessing-bdai) [reply msg/KeyValueBosdynMsgsSerializedMessage.msg]: Fixed that issue

**Review by jbarry-bdai — APPROVED** [review]
We also need to create a new release for this... see the README.

**Comment by amessing-bdai** [comment]
Yes, I’ve already rebuilt the Debian. About to test it then will make the
release

**Pull request merged**